README.md

```diff
@@ -59,14 +63,12 @@ class Restaurant < ActiveRecord::Base
     has_many :dishes, through: "Restaurant__c"
     belongs_to :chef, through: %w(Chef__c Cuisine__c)
 
+    converts_with StyleAdapter
```

Comment:
I really didn't want to get too fancy with this. We can just extend the default `Restforce::DB::Adapter` class if we need/want to maintain the default functionality for a custom adapter.